cmd/ocitools/generate.go

```diff
@@ -305,61 +311,81 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 		}
 	}
 
-	var sd string
-	var sa, ss []string
+	err := addSeccomp(*g, context)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func addSeccomp(g generate.Generator, context *cli.Context) error {
```

Comment:
To stick with the new-since-around-#143 pattern, I'd rather see `*generate.Generator` here too.